CHANGELOG.latest.md

```diff
@@ -1,5 +1,2 @@
-### Breaking Changes
-* iOS rename `productCategory` to `product type` and `productType` to `productSubtype` (#377) via Josh Holtz (@joshdholtz)
-* Android rename `productCategory` to `productType` and `productType` to `productSubtype` (#376) via Josh Holtz (@joshdholtz)
-### Other Changes
-* Use new mapped product type values for `purchaseProduct()` and `getProductInfo()` (#384) via Josh Holtz (@joshdholtz)
+### Bugfixes
+* Add platform check,`OfferPaymentMode`, and `presentedOfferingIdentifier` (#371) via Josh Holtz (@joshdholtz)
```

Comment:
Same for this changelog. Maybe it's better to not revert this, since latest release is still the same. But it's ok if you want to fix that in another PR

Reply:
Ahhh, good point! I'll just revert the 3 main PRs 💪